lib/common/options.js

```diff
@@ -4,6 +4,7 @@ import fs from 'fs'
 import _ from 'lodash'
 import consola from 'consola'
 
+import compressionMiddleware from 'compression'
```

Comment:
I've renamed it :ok_hand: 

Could you explain how you mean *lazy loading* it?

In the end, it's "just" the server bundle anyway.

Reply:
require or nuxt.requireModule inside renderer so that we can support sth like: compressor: ‘custom-lib’ or compressor: { name: ‘compression’, options: {}}. Maybe I thought too much, current pr is good for me. Let’s ask @pi0 review and merge.